opal/mca/btl/scif/btl_scif_component.c

```diff
@@ -0,0 +1,387 @@
+/* -*- Mode: C; c-basic-offset:4 ; indent-tabs-mode:nil -*- */
```

Comment:
I don't understand how this file got in as the entire scif component has been removed from OMPI.

Reply:
this is a mistake when i rebase, one commit got this added, already deleted